src/Microsoft.DotNet.XHarness.CLI/Commands/iOS/iOSTestCommand.cs

```diff
@@ -76,7 +77,7 @@ protected override async Task<ExitCode> InvokeInternal()
             var processManager = new ProcessManager(_arguments.XcodeRoot, _arguments.MlaunchPath);
             var deviceLoader = new HardwareDeviceLoader(processManager);
             var simulatorLoader = new SimulatorLoader(processManager);
-            var tunnelBore = new TunnelBore(processManager);
+            var tunnelBore = (_channel == CommunicationChannel.UsbTunnel) ? new TunnelBore(processManager) : null;
```

Comment:
I think it would be better to have `Channel` in `_arguments` so that it is together with the rest.

I think later the whole `OptionSet` will actually move to `Arguments` (when we do nullability). It also makes more sense responsibility wise. So we would forget something behind like this.